### DISCUSSION TITLE
Latest Update - IP Address?

### DISCUSSION THREAD

**Discussion by MopedLampe** [UnchartedBull/OctoDash#2135] [Q&A] [answered]
Hello. 

First of all, thank you for octodash. I love it.
Since i read about that with the latest update there is the possibility to see the Raspis IP address, can you tell me where?
I still cant see it and it would be great if i could. ;)

Thank you all

**Comment by UnchartedBull** [comment]
There currently is no option to display the IP on OctoDash. Sorry.

**Comment by queengooborg** [comment, marked as answer]
This has recently been added and will be available in v2.4.0!

**Reply by jneilliii** [reply]
from your forked release, the IP address for me in about shows as 127.0.0.1.

**Reply by queengooborg** [reply]
Strange, the IP was properly reported for me on my Pi -- mind filing an issue?

**Reply by jneilliii** [reply]
I will if the issue persists after the next official release over here.
